runtime/bcmath.cpp

```diff
@@ -326,7 +326,7 @@ static string bc_mul_positive(const char *lhs, int lint, int ldot, int lfrac, in
 
   dl::deallocate(res, static_cast<size_t>(sizeof(int) * result_size));
 
-  return bc_round(result.buffer(), resint, resdot, resfrac, resscale, scale, sign, 0);
+  return bc_round(result.buffer(), resint, resdot, resfrac, resscale, scale, sign, 1);
```

Comment:
In `bc_div_positive` this is left 0, as you've changed to 1 only here, in `bc_mul_positive`. Shouldn't div also be modified? (I mean, we need to nest how PHP works)